package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "tacos",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "seed:dev": "nodemon server.js",
```

Comment:
The newest version of node just have `--watch` which is nice so you don't need this.